pkg/actions/param_delete_test.go

```diff
@@ -27,29 +27,56 @@ import (
 )
 
 func TestParamDelete(t *testing.T) {
-	withApp(t, func(appMock *amocks.App) {
-		componentName := "deployment"
-		path := "replicas"
-
-		c := &cmocks.Component{}
-		c.On("DeleteParam", []string{"replicas"}).Return(nil)
-
-		in := map[string]interface{}{
-			OptionApp:  appMock,
-			OptionName: componentName,
-			OptionPath: path,
-		}
-
-		a, err := NewParamDelete(in)
-		require.NoError(t, err)
-
-		a.resolvePathFn = func(app.App, string) (component.Module, component.Component, error) {
-			return nil, c, nil
-		}
-
-		err = a.Run()
-		require.NoError(t, err)
-	})
+	cases := []struct {
+		name  string
+		isErr bool
+	}{
+		{
```

Comment:
How do these two test cases differ?

Reply:
Updated the test case to be more obvious about the difference. Squashing